assets/app/scripts/directives/popups.js

```diff
@@ -27,11 +30,14 @@ angular.module('openshiftConsole')
               // If dynamic-content attr is set at all, even if it hasn't evaluated to a value
               if (attrs.dynamicContent || attrs.dynamicContent === "") {
                 $scope.$watch('dynamicContent', function() {
-                  $(element)
-                    .attr("title", $scope.dynamicContent)
-                    .tooltip("destroy")
-                    .tooltip();                
-                });                  
+                  $(element).tooltip("destroy");
+                  // Destroy is asynchronous. Wait for it to complete before updating content.
```

Comment:
update the comment to refer to the known hide timeout https://github.com/twbs/bootstrap/blob/87121181c8a4b63192865587381d4b8ada8de30c/js/tooltip.js#L31